### PR TITLE
Imperial Cycle

### PR DESCRIPTION
**Pull request by mplain**

### CODE REVIEW

**Comment by Alsciende** [comment]
Thanks!

File: Card/kakita-yoshi.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        "political": "6",
+        "political_bonus": null,
+        "role_restriction": null,
+        "side": "dynasty",
+        "strength": null,
+        "strength_bonus": null,
+        "text": "<b>Action:</b> While this character is participating in a conflict, discard the Imperial Favor – draw 3 cards. Reduce the cost of each event you play during this conflict by 2.",
+        "traits": [
+            "courtier",
+            "daimyō",
```
Comment (Alsciende) [inline Card/kakita-yoshi.json]: @mplain traits must be lowercase ascii, like "id" -> "daimyo"

File: Card/talisman-of-the-sun.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+        "political": null,
+        "political_bonus": "+1",
+        "role_restriction": "keeper",
+        "side": "conflict",
+        "strength": null,
+        "strength_bonus": null,
+        "text": "<em>Keeper</em> role only.\n<b>Action:</b> During a conflict in which you are the defending player, bow this attachment – move the contested ring to another eligible province and reveal it, if able. That province is now the attacked province.",
+        "traits": [
+            "item",
+            "meishōdō"
```
Comment (Alsciende) [inline Card/talisman-of-the-sun.json]: ascii

File: Card/kakita-yoshi.json (hunk 1 above)
Comment (mplain) [reply Card/kakita-yoshi.json]: Got it!

**Pull request merged**